csharp/ql/test/library-tests/ir/offbyone/test.cs

```diff
@@ -0,0 +1,104 @@
+class ContainerLengthOffByOne
+{
+    public int[] arr;
+    public string str;
+
+    public static void fun(int elem)
+    {
+    }
+
+    public void test1() 
```

Comment:
Tiny nit but we tend to use capitals, i.e. `Test1`

Reply:
Fixed